packages/suite/src/components/dashboard/NewsFeed/index.tsx

```diff
@@ -1,20 +1,25 @@
 import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
+import axios from 'axios';
```

Comment:
regular `fetch` isn't enough? see newsletterActions

Reply:
btw. this 'axios' package is not a part of `@suite:package.json`
imho we shouldn't use it here, in `@suite`